packages/terminal/src/index.ts

```diff
@@ -25,3 +24,88 @@ export const ITerminalTracker = new Token<ITerminalTracker>(
   '@jupyterlab/terminal:ITerminalTracker'
 );
 /* tslint:enable */
+
+/**
+ * The namespace for terminals.
+ */
+export namespace ITerminal {
+  export interface ITerminal extends Widget {
+    /**
+     * The terminal session associated with the widget.
+     */
+    session: TerminalSession.ISession | null;
+
+    /**
+     * Get a config option for the terminal.
+     */
+    getOption<K extends keyof IOptions>(option: K): IOptions[K];
+
+    /**
+     * Set a config option for the terminal.
+     */
+    setOption<K extends keyof IOptions>(option: K, value: IOptions[K]): void;
+
+    /**
+     * Refresh the terminal session.
+     */
+    refresh(): Promise<void>;
+  }
+  /**
+   * Options for the terminal widget.
+   */
+  export interface IOptions {
+    /**
+     * The font family used to render text.
+     */
+    fontFamily: string | null;
+
+    /**
+     * The font size of the terminal in pixels.
+     */
+    fontSize: number;
+
+    /**
+     * The line height used to render text.
+     */
+    lineHeight: number | null;
+
+    /**
+     * The theme of the terminal.
+     */
+    theme: Theme;
+
+    /**
+     * The amount of buffer scrollback to be used
+     * with the terminal
+     */
+    scrollback: number | null;
+
+    /**
+     * Whether to blink the cursor.  Can only be set at startup.
+     */
+    cursorBlink: boolean;
+
+    /**
+     * An optional command to run when the session starts.
+     */
+    initialCommand: string;
+  }
+
+  /**
+   * The default options used for creating terminals.
+   */
+  export const defaultOptions: IOptions = {
+    theme: 'dark',
+    fontFamily: 'Menlo, Consolas, "DejaVu Sans Mono", monospace',
```

Comment:
Had to do this, so i get my beautiful black-and-white emoji with broken character widths... also looks better on windows.